src/org/parosproxy/paros/model/SiteNodeStringComparator.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development Team
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.parosproxy.paros.model;
+
+import java.util.Comparator;
+
+/**
+ * Based on example code from:
+ * @link http://www.java2s.com/Code/Java/Swing-JFC/AtreemodelusingtheSortTreeModelwithaFilehierarchyasinput.htm
+ */
+class SiteNodeStringComparator implements Comparator<SiteNode> {
+	public int compare(SiteNode sn1, SiteNode sn2) {
+		if (!(sn1 instanceof SiteNode && sn2 instanceof SiteNode)) {
+			throw new IllegalArgumentException("Can only compare SiteNode objects");
+		}
+		String s1 = sn1.getCleanNodeName();

```

Comment:
This should use `getNodeName()` instead as that ensures that the sorting is done on what is shown (i.e. includes the method and parameters)?